src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToArrayTransformer.php

```diff
@@ -185,9 +185,10 @@ public function reverseTransform($value)
                 empty($value['day']) ? '1' : $value['day'],
                 empty($value['hour']) ? '0' : $value['hour'],
                 empty($value['minute']) ? '0' : $value['minute'],
-                empty($value['second']) ? '0' : $value['second'],
-                $this->outputTimezone
-            ));
+                empty($value['second']) ? '0' : $value['second']
++               ),
++new \DateTimeZone($this->outputTimezone)
```

Comment:
well, I know, but fabbot.io suggested it - I have applied it with 
curl http://fabbot.io/patch/symfony/symfony/15163/b8363efb1fdc0633b32e41c354b90c1a54f90ef9/cs.diff | patch -p0

That page is not showed now